kotlin-website.py

```diff
@@ -236,6 +236,10 @@ def kotlin_reference_pdf():
 def kotlin_docs_pdf():
     return send_file(path.join(root_folder, "assets", "kotlin-reference.pdf"))
 
+@app.route('/docs/<path:path>')
```

Comment:
should we add some conditions here to exclude such routing on production? @zoobestik

Reply:
added check if it is prodcution